foundation/foundation-mda/src/main/resources/profiles.json

```diff
@@ -542,9 +542,6 @@
 			},
 			{
 				"name": "mlflowLineageProperties"
-			},
-			{
-				"name": "trainingGitkeepFile"
```

Comment:
Q: What is this removing?

Reply:
because the directory is not empty so we don't need the keep file there